android/src/main/kotlin/com/hoanglm/flutteryoutubeview/FlutterYoutubeViewPlugin.kt

```diff
@@ -65,7 +69,9 @@ class FlutterYoutubeViewPlugin(registrar: Registrar): Application.ActivityLifecy
         @JvmStatic
         fun registerWith(registrar: Registrar) {
```

Comment:
Hi @Henry-Keys ,
When app running in background, I think no need register for this plugin. Please see my solution in below. Is it Ok? 
Thanks so much
```
companion object {
        @JvmStatic
        fun registerWith(registrar: Registrar) {
            if (registrar.activity() == null) {
                // When a background flutter view tries to register the plugin, the registrar has no activity.
                // We stop the registration process as this plugin is foreground only.
                return;
            }
            val plugin = FlutterYoutubeViewPlugin(registrar)
            registrar.activity().application.registerActivityLifecycleCallbacks(plugin)
            registrar
                .platformViewRegistry()
                .registerViewFactory(
                    "plugins.hoanglm.com/youtube", YoutubeFactory(registrar, plugin.state)
                )
        }
    }
```

Reply:
That's simpler. We can stay with that.
Thanks.

Reply:
Thanks. I will release soon